items.json

```diff
@@ -8553,9 +8553,9 @@
     "name": "Early Retirement",

```

Comment:
Same here , have in mind that it will be displayed in onehanded

Reply:
removed 1h modes from most weps